spec/ddtrace/contrib/http/request_spec.rb

```diff
@@ -334,4 +334,35 @@ def expect_request_without_distributed_headers
       end
     end
   end
+
+  describe 'request exceptions' do
+    subject(:response) do
+      begin
+        client.get(path)
+      # rubocop:disable Lint/UselessAssignment
+      rescue => e
+        nil
+      end
+    end
+
+    let(:path) { '/my/path' }
+
+    context 'that raise a timeout exception' do
+      before(:each) { stub_request(:any, "#{uri}#{path}").to_timeout }
+
+      let(:span) { spans.first }
+
+      it 'generates a well-formed trace with span tags available from request object', :focus do
```

Comment:
I think you should remove `:focus` otherwise it might only run this one test.

Reply:
whoops, leftover from dev, apologies. updated.